src/query/storages/parquet/src/parquet_rs/parquet_reader/utils.rs

```diff
@@ -119,7 +121,7 @@ pub fn compute_output_field_paths(
     if !inner_projection {
         return Ok(None);
     }
-    let expected_schema = to_arrow_schema(expected_schema);
+    let expected_schema = DataSchema::from(expected_schema);
```

Comment:
@b41sh 
so we infact get the right schema here,  what this pr does is to pass it to needed place.
am I right?

Reply:
yes, the schema in `RecordBatch` don't have extension type